extensions/classifiers/svm-prediction.service.spec.ts

```diff
@@ -143,5 +143,89 @@ describe('SVM prediction functions', () => {
         ).toBeLessThan(1e-3);
       }
     });
+
+    it('should throw error when dimension of support' +
+      'vectors and given input is different.', () => {
+      const errorLog = spyOn(console, 'error').and.callThrough();
+      var svmData = {
+        classes: [0, 1],
+        kernel_params: {
+          kernel: 'linear', coef0: 0.0, degree: 3, gamma: 0.5},
+        intercept: [0.04554340162799716],
+        n_support: [80, 66],
+        probA: [-4.76812258346006],
+        support_vectors: [
+          [5.1, 0.0], [4.0, -2.0], [3.0, -4.0], [-5.0, 0.0], [4.0, -2.0],
```

Comment:
Do you need such a big test case? I feel if you create a smaller test case it becomes easier to understand the test you wrote..